package-schema-registry/test/fixtures/cluster-bridged-plain.yml

```diff
@@ -0,0 +1,120 @@
+---
+version: '2'
+networks:
+  zk:
```

Comment:
the naming here is a bit confusing since the network is used by everything, not just zk.